pkg/csi/fuse/nodeserver.go

```diff
@@ -155,13 +163,138 @@ func (ns *nodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 }
 
 func (ns *nodeServer) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstageVolumeRequest) (*csi.NodeUnstageVolumeResponse, error) {
+	// 1. get fuse label according to pv namespace and name
+	namespacedName := strings.Split(req.GetVolumeId(), "-")
+	fuseLabelKey := common.LabelAnnotationFusePrefix + namespacedName[0] + "-" + namespacedName[1]
+
+	// 2. check if the path is mounted
+	inUse, err := checkMountInUse(req.GetVolumeId())
+	if err != nil {
+		return nil, errors.Wrap(err, "NodeUnstageVolume: can't check mount in use")
+	}
+	if inUse {
+		return nil, fmt.Errorf("NodeUnstageVolume: can't stop fuse cause it's in use")
+	}
+
+	// 3. remove label on node.
+	// Once the label is removed, fuse pod on corresponding node will be terminated
+	// since node selector in the fuse daemonSet no longer matches.
+	err = retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```

Comment:
I have a suggestion that:
1. adding and removing label could consider  `patch`
2. adding and removing the same label should consider synchronization
3. Use a struct named `NodeLabel` with two methods add and remove labels  .